mxcube3/ui/components/Tasks/Characterisation.js

```diff
@@ -1,9 +1,7 @@
-'use strict';
-
 import React from 'react';
 import { reduxForm } from 'redux-form';
 import { Modal } from 'react-bootstrap';
-
+/* eslint camelcase: 0 */
```

Comment:
I understand that this is because of the back end that exports variables with _. Perhaps its also a temporary fix ?. Id rather see that we do some kind of conversion/mapping than disabling the style guide.

Another thing I noted is that you guys seem to mix upper and lower camel case, like point and Type in the code below. It does not matter to me which one we use as long as we are consistent. We should however use lower camel case if we want to respect the style guide.